libs/webb-ui-components/src/components/WebsiteFooter/WebsiteFooter.tsx

```diff
@@ -13,16 +15,110 @@ import {
   Typography,
 } from '@webb-tools/webb-ui-components';
 import Link from 'next/link';
-import { type } from 'os';
 import { ComponentProps, useState } from 'react';
-import { WebsiteNewletterForm } from '../WebsiteNewsLetterForm';
+import { WebsiteNewsletterForm } from '../WebsiteNewsLetterForm';
+
+type NavLinkType = {
+  group: string;
+  links: Array<
+    ComponentProps<typeof InternalOrExternalLink> & { label: string }
+  >;
+};
 
 type SocialType = {
   Icon: (props: IconBase) => JSX.Element;
   name: string;
   href: string;
 };
 
+const navLinks: Array<NavLinkType> = [
```

Comment:
Can we make the following changes:

- [ ] Replace "Resources" with "Community" and include the same links as the docs site
- [ ] Can we replace "Network" with "Ecosystem", include the links as the docs site, and remove "DAPP" column
- [ ] Add "Legal" column and include the privacy and terms links